### ISSUE TITLE
Faster Training Process for MONAI Label

### ISSUE THREAD

**Issue by finalelement** [Project-MONAI/MONAILabel#444]
This is a list of techniques for faster convergence to complete the training process faster for Spleen/Heart.

Experimental Settings observed for trial are:
- Novograd Optimizer instead of Adam
- Higher learning rate of 0.0002 vs 0.0001
- Dice CE Loss with squared predictions and batch set to true

@SachidanandAlle If you have another issue in parallel for the thread data loader, please direct me to the correct milestone and Ill tag this one there as well. 

I will report here with experimental findings.

**Comment by SachidanandAlle** [comment]
can you prepare a PR for these changes?

**Comment by finalelement** [comment]
PR raised: https://github.com/Project-MONAI/MONAILabel/pull/485

**Comment by SachidanandAlle** [comment]
I am working on this.. will add options to select ThreadDataLoader and provision user to run some transforms on GPU

**Comment by SachidanandAlle** [comment]
https://github.com/Project-MONAI/MONAILabel/pull/485 solves this problem

For example the training per epoch is reduced to 4 sec from 16 sec (keeping the hardware constant)
